samples/drivers/led_ws2812/src/main.c

```diff
@@ -21,7 +21,7 @@ LOG_MODULE_REGISTER(main);
 #define STRIP_NODE		DT_ALIAS(led_strip)
 #define STRIP_NUM_PIXELS	DT_PROP(DT_ALIAS(led_strip), chain_length)
 
-#define DELAY_TIME K_MSEC(50)
+#define DELAY_TIME K_MSEC(CONFIG_LED_UPDATE_DELAY)
```

Comment:
needs renaming too -- sorry for the extra work for what you thought would be a trivial change 😑

Reply:
My bad. Should be fixed now.